core/src/View/MDSpan/Kokkos_MDSpan_Layout.hpp

```diff
@@ -111,6 +111,9 @@ KOKKOS_INLINE_FUNCTION auto array_layout_from_mapping(
                        rank > 6 ? ext.extent(6) : KOKKOS_IMPL_CTOR_DEFAULT_ARG,
                        rank > 7 ? ext.extent(7) : KOKKOS_IMPL_CTOR_DEFAULT_ARG};
   }
+#ifdef KOKKOS_COMPILER_INTEL
```

Comment:
Does this not also cause problems on NVCC? I seem to recall they share the same frontend and both h ave this bug

Reply:
I looked at one of the nightly Cuda jobs and did not see any warnings related to mdspan